2023/did-wg.html

```diff
@@ -180,6 +180,7 @@ <h2>Scope</h2>
 
         <p>
 The Working Group may also publish new Working Group Notes.
+Notes MUST NOT include submissions for DID Methods and MUST NOT address issues regarding any particular DID Method. The WG welcomes DID Method standardization outside the WG (whether in the W3C or in other standards bodies).
```

Comment:
```suggestion
Notes will not include DID Method specifications and will not address issues regarding particular DID Methods. The WG welcomes DID Method standardization outside the WG (whether in the W3C or in other standards bodies).
```

Reply:
This is an attempt to address @brentzundel's concern about normative language.

Since this is about the scope of the group, we don't need MUST and such.  

However, I do have a concern that even the language I propose would mean we could not maintain the Rubric as a NOTE. It is expected to shift to a registry, but we should clarify how this language affects that publication.
```suggestion
Any additional notes beyond those explicitly listed in the deliverable section will not include DID Method specifications and will not address issues regarding any particular DID Methods. The WG welcomes DID Method standardization outside the WG (whether in the W3C or in other standards bodies).
```